letsdrill.rb

```diff
@@ -1,24 +1,42 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
+
+  case integer
+  when 90..100
+    return 'A'
+  when 80..90
+    return 'B'
+  when 70..80
+    return 'C'
+  when 60..70
+    return 'D'
+  when 0..60
+    return 'F'
+  end
 
 end
 
 def shortest_string(array)
 
-  #Put your code here!
+  short_string = array.first
+
+  array.each do |str|
+    if str.length < short_string.length
+      short_string = str
+    end
+  end
 
+  short_string
 end
- 
```

Comment:
This works, but there are Ruby built-in methods that already do the same thing much more succinctly. Check out the Enumerable module, specifically the parts on sort_by and min_by. [link](http://ruby-doc.org/core-2.2.3/Enumerable.html)